lisp/ess-r-completion.el

```diff
@@ -367,6 +367,29 @@ To be used instead of ESS' completion engine for R versions >= 2.7.0."
 ;;; COMPANY
 ;;; http://company-mode.github.io/
 
+(defvar company-R-library--cache ())
```

Comment:
I think `installed.packages` has a built-in cache mechanism.

There is also `.packages` function. I wonder which one would be better for this purpose.

Reply:
How would you check for non-local libPaths. What would they look liike? I'm sure there is a function, but I can't find documentation on it.

Reply:
I don't actually know and I am pretty sure those would look differently depending on the system or network path type. 

Let's go with this as it and see how it goes. We can add this to the default backends and see if people complain. If this change causes serious slowdown even on local machines then, I am afraid, we will have to keep it outside of default backends.

Reply:
even if you had every package on cran it still seems like it wouldn't be slow